src/Raven.Server/Config/RavenConfiguration.cs

```diff
@@ -181,8 +181,8 @@ public void PostInit()
 
         public void CopyParentSettings(RavenConfiguration serverConfiguration)
         {
-            Encryption.UseSsl = serverConfiguration.Encryption.UseSsl;
-            Encryption.UseFips = serverConfiguration.Encryption.UseFips;
+            Security.CertificatePassword = serverConfiguration.Security.CertificatePassword;
+            Security.CertificateFilePath = serverConfiguration.Security.CertificateFilePath;
```

Comment:
Under what scenario do we need to copy the parent configuration and use those values not on the server level?